### ISSUE TITLE
Could not use script templates with table-rows

### ISSUE THREAD

**Issue by Boeller666** [ErikSchierboom/knockout-pre-rendered#42]
There is an issue using table-row templates in script-tags.

Changing code on line 77 following to code below will fix it:
```javascript
if (sourceNode.content) {
    // For e.g. <template> tags
      parentNode = sourceNode.content;
    } else if (sourceNode.tagName === 'SCRIPT') {
      if (sourceNode.innerHTML.match(/<tr[\s\S]*?<\/tr>/g)) {
        parentNode = document.createElement('tbody');
      }
      else {
        parentNode = document.createElement('div');
      }
      parentNode.innerHTML = sourceNode.innerHTML;
    } else {
      // Anything else e.g. <div>
      parentNode = sourceNode;
    }
```

After this change a template like this:
```html
        <script type="text/ko-template" id="taskTemplate">
            <tr data-bind="attr: { id: id }">
                <td data-bind="template: $root.getDescriptionCellTmpl">
                </td>
                <td data-bind="template: { name: 'userTemplate', foreach: users }">
                </td>
            </tr>
        </script>
```
will work

**Comment by ErikSchierboom** [comment]
I'll look into it soon! Thanks for the detailed report.

**Comment by ErikSchierboom** [comment]
@Boeller666 I've modified the code (as suggested). I have placed the code in a branch. Would you be so kind to verify if the change has the intended effect? You can find the file here: https://raw.githubusercontent.com/ErikSchierboom/knockout-pre-rendered/tr-tags-script/dist/knockout-pre-rendered.js

**Comment by ErikSchierboom** [comment]
@Boeller666 Or perhaps create a somewhat more extensive repro?

**Comment by boedlen** [comment]
Hi @ErikSchierboom, i ran into the same issue with table rows not working correctly, and i've tried your modified code and can confirm that it fixed the issue for me.

**Comment by ErikSchierboom** [comment]
@boedlen Thanks for the feedback! I've just merged the change and pushed a new version (0.7.1) to NPM. It is also available from Bower.
